src/pykeen/models/predict.py

```diff
@@ -179,7 +179,8 @@ def get_prediction_df(
     )
 
     # get scores
-    scores = model.predict(batch, full_batch=False, mode=mode, ids=targets, target=target).squeeze(dim=0).tolist()
+    with torch.no_grad():
```

Comment:
there is also [`torch.inference_mode`](https://pytorch.org/docs/stable/generated/torch.inference_mode.html)

Moreover, you can use that as a annotation of the whole `get_prediction_df` function, i.e.,
```python
@torch.inference_mode()
def get_prediction_df(...)
```